src/components/interpolation.js

```diff
@@ -54,8 +54,14 @@ function onlyHasDefaultPlace (params) {
 
 function useLegacyPlaces (children, places) {
   const params = places ? createParamsFromPlaces(places) : {}
+  
   if (!children) { return params }
 
+  // Filter empty text nodes
+  children = children.filter(child => {
+    return child.tag || (child.text = child.text.trim())
```

Comment:
This filtering uses assignment which is not clear, I think.
How about this ?
```suggestion
    return child.tag || child.text.trim() !== ''
```

Reply:
I'm not sure why it used assignment. I copied it from pre-#685 code. Checking blame right now.

Reply:
Looks like it wasn't needed there too